mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/persistence/MantisStorageProviderAdapter.java

```diff
@@ -50,6 +66,27 @@
 public class MantisStorageProviderAdapter implements IMantisStorageProvider {
```

Comment:
This looks like KVDal specific implementation. I would maybe rename this to KVDalAwareMantisStorageProvider

Reply:
done